tests/integration/test_subscriptions_cli.py

```diff
@@ -133,3 +133,114 @@ def test_subscriptions_bad_request(cmd_arg, runner_input):
 
     assert result.exit_code == 2  # bad parameter.
     assert "Request does not contain valid json" in result.output
+
+
+def test_subscriptions_cancel_failure(monkeypatch):
+    """Cancel command exits gracefully from an API error."""
+
+    monkeypatch.setattr(planet.cli.subscriptions, '_fake_subs', {})
```

Comment:
Tweaking the state of the API for this new test. It's now a dictionary, where keys are subscription ids and values are subscriptions.